control_apps/src/control_mngr.rs

```diff
@@ -8,6 +8,8 @@ use std::fmt;
 use std::thread::sleep; // only used to emulate some sort of POST
 use std::time::Duration;
 
+use log::{info, warn, debug};
```

Comment:
no error log? is that because there are no error statements in this crate?

Reply:
yes